### PR TITLE
Create test that tests custom form with IEF element and make it pass.

### PR DESCRIPTION
**Pull request by slashrsm**

### CODE REVIEW

File: src/Element/InlineEntityForm.php [hunk 1]
```diff
@@ -138,9 +115,142 @@ public static function processEntityForm($element, FormStateInterface $form_stat
...
+    $submit_attached = FALSE;
+    $submit = array_merge([[get_called_class(), 'triggerIefSubmit']], $complete_form['#submit']);
+
+    if (!empty($complete_form['submit'])) {
+      $complete_form['submit']['#submit'] = empty($complete_form['submit']['#submit']) ? $submit : array_merge($submit, $complete_form['submit']['#submit']);
+      $complete_form['submit']['#ief_submit_all'] = TRUE;
+      $submit_attached = TRUE;
+    }
+
+    if (!empty($complete_form['actions']['submit'])) {
```
Comment (marcingy) [inline src/Element/InlineEntityForm.php]: The next 3 could be a foreach so ['submit', 'publish', 'unpublish']?

File: src/Element/InlineEntityForm.php [hunk 2]
```diff
@@ -138,9 +115,142 @@ public static function processEntityForm($element, FormStateInterface $form_stat
...
+      $complete_form['actions']['unpublish']['#ief_submit_all'] = TRUE;
+      $submit_attached = TRUE;
+    }
+
+    // If we didn't attach submit to one of the most common buttons let's search
+    // the form for any submit with #button_type == primary and attach to that.
+    if (!$submit_attached) {
+      static::recurseAttachMainSubmit($complete_form, $submit);
+    }
+
```
Comment (marcingy) [inline src/Element/InlineEntityForm.php]: Extra blank line

**Comment by webflo** [comment]
Enabled Travis CI integration on this repo. Please add a config file for travis ci if the test is green.

**Comment by slashrsm** [comment]
Implemented in https://github.com/drupal-media/entity_browser/pull/56

**Comment by slashrsm** [comment]
Green :)

**Pull request merged**